charts/terra-prometheus/templates/rules/appRules.yaml

```diff
@@ -118,11 +118,13 @@ spec:
       for: 10m
       labels:
         severity: warning
+    {{- if .Values.enableCronjobAlerts }}
     - alert: KubeJobFailed
       annotations:
         message: Job {{`{{`}} $labels.namespace {{`}}`}}/{{`{{`}} $labels.job_name {{`}}`}} failed to complete.
         runbook_url: alert-name-kubejobfailed
       expr: kube_job_status_failed{job="kube-state-metrics"}  > 0
```

Comment:
any way we can suppress alerting for just the broken Leo cron jobs? Wondering if it would be better to disable them in perf in the Leo chart instead of disabling alerting for all cron jobs in perf...

Reply:
Perhaps the best option here is to use labels to more specifically target the leo crons. Let me see if I can come up with a better approach